examples/__helpers__/emptyRedrawQueue.js

```diff
@@ -0,0 +1,6 @@
+module.exports.emptyRedrawQueue = async () => {
+    await page.waitFor(
+        () =>
+            document['__d3fc-elements__'] == null
```

Comment:
Should this use `packages\d3fc-element\src\data.js\key`, or at least move that functionality to this examples folder?

Reply:
I can't access `key` as it is not currently exported from that module nor from the package (`packages/d3fc-element/index.js`). I can't see a good reason other than testing to access the value so I'd rather not change that.

As this is test code, if the value ever changes then the tests will fail so I'm happy to have the duplication

Reply:
Sounds good.